app/src/main/kotlin/com/ayatk/biblio/data/repository/RankingRepositoryImpl.kt

```diff
@@ -53,29 +53,29 @@ class RankingRepositoryImpl @Inject constructor(
       }
 
       return narouDataStore.getRanking(apiRankingType, today.time)
-          .flatMap {
-            val codes = it
-                .map { it.ncode }
-                .drop(range.first)
-                .take(range.count())
+        .flatMap {
+          val codes = it
+            .map { it.ncode }
+            .drop(range.first)
+            .take(range.count())
 
-            val query = QueryBuilder()
-                .ncode(*codes.toTypedArray())
-                .size(range.count())
-                .build()
+          val query = QueryBuilder()
+            .ncode(*codes.toTypedArray())
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Using a spread operator causes a full copy of the array to be created before calling a method has a very high performance penalty.